### ISSUE TITLE
UC Browser Compatibility Problem

### ISSUE THREAD

**Issue by bdthemes** [uikit/uikit#2669]
Hi, When i check from UC Browser from mobile layout not showing correctly. please check getuikit.com in your UC browser from any android phone.

### UIkit version

3.0.0-beta.22

### Browser
UC Browser 11.3.0.950 - Mobile

### Reproduction Link

https://getuikit.com

### Steps to reproduce
Just check https://getuikit.com

### What is Expected?
Nothing properly work as i see on desktop or others mobile browser

### What is actually happening?
Look those screenshot.
http://prntscr.com/f7a6xl
http://prntscr.com/f7a769

**Comment by bdthemes** [comment]
@florianletsch I know the UIkit browser compatibility list but UC Browser also a popular browser for mobile it's already using 100000000+ user ( check this link:
https://play.google.com/store/apps/details?id=com.UCMobile.intl&hl=en ) so i think you should give support this browser otherwise they missing uikit 3 based website :( 

It's **U3 engine (based on [Webkit](https://en.wikipedia.org/wiki/WebKit))** for more details please check here: https://en.wikipedia.org/wiki/UC_Browser

**Comment by florianletsch** [comment]
You are right, UC Browser has a considerable market share, I honestly wasn't aware of that. 

Thanks, we'll look into this.
